tests/usage.test.ts

```diff
@@ -41,11 +43,19 @@ const schemas = [
     schema: applicationSchema,
     examples: getJSONExamples<Application>('application'),
   },
+  // {
+  //   name: 'PrototypeApplication',
+  //   schema: prototypeApplicationSchema,
+  //   examples: getJSONExamples<PrototypeApplication>('prototypeApplication'),
+  // },
```

Comment:
This didn't work so smoothly & throws a number of type errors ! 

One to sort out in a follow-up PR I think, and for here I'm just relying on TS definitions & `pnpm build-json-examples` as the "validation" check (eg building the JSON example fails if type errors).